### PR TITLE
Parser for the metadata.

### PR DESCRIPTION
**Pull request by RaviShivam**
Parses specifications of a certain specimen and stores into hash map.

### CODE REVIEW

File: src/main/java/io/github/programminglife2016/pl1_2016/parser/Data.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package io.github.programminglife2016.pl1_2016.parser;
+
+/**
+ * Created by ravishivam on 4-5-16.
+ */
+public interface Data {
+}
```
Comment (smesseim) [inline src/main/java/io/github/programminglife2016/pl1_2016/parser/Data.java]: An interface without methods is usually not what you want. Ideally every method of the implementations of an interface should override the interface methods. Also, `Data` is not a clear name.

**Comment by smesseim** [comment]
Is `Specimen` a replacement for `Segment`? What exactly is the purpose of `Specimen`?

**Comment by RaviShivam** [comment]
Specimen is NOT a replacement for segment. A specimen is whom a certain genome belongs to. The genome is contructed by a serie of segments. Thus segments are pieces of a genome that belongs to a certain specimen. The specimen class tells something about the person that the genome belongs to i.e. age, gender, HIV status etc.

**Comment by smesseim** [comment]
Is `Subject` already linked to the Nodes?

**Comment by RaviShivam** [comment]
Yes. It is. Because Segment now contains a List with specimen (genomes) it belongs to. And that can directly be linked to a certain specimen. The name of the specimen (format: TKxxxx) is the key with which you can access all the information of that specimen in the SpecimenMap class

**Comment by smesseim** [comment]
LGTM

**Comment by SamuelSital** [comment]
LGTM

**Pull request merged**